### ISSUE TITLE
Fix flaky zombienet tests `zombienet-polkadot-coretime-revenue`

### ISSUE THREAD

**Issue by pepoviola** [paritytech/polkadot-sdk#5848]
Example: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/7444000 (fails)
Retry works: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/7444917

cc: @s0me0ne-unkn0wn

**Comment by s0me0ne-unkn0wn** [comment]
From the logs, I can only guess that the parachain never starts producing blocks.
It happened to me sometimes in local tests, and it seemed to me much more like a Zombienet problem. I hoped it was connected with the native provider and would be gone after switching to k8s.
It would be interesting to have a look into the collator logs.

**Comment by pepoviola** [comment]
Hi @s0me0ne-unkn0wn, thanks for looking this.  I checked the logs and ehe collator seems producing blocks ok. 
https://grafana.teleport.parity.io/goto/Z5XCkjgNR?orgId=1

Thx!

**Comment by s0me0ne-unkn0wn** [comment]
Nope, the relay chain side is producing but the parachain has stuck at the best block №3... I'll try to look into that in the next days.

**Comment by pepoviola** [comment]
> Nope, the relay chain side is producing but the parachain has stuck at the best block №3... I'll try to look into that in the next days.

Ahh my bad, I quickly look and don't realize of the stuck. Thanks!

**Comment by alindima** [comment]
I see something similar occasionally on my new zombienet-sdk tests

**Comment by alindima** [comment]
I see a ton of similar failures to spawn lately on the zombienet-sdk tests @pepoviola

**Comment by alindima** [comment]
Some examples:

https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/7932767
https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/7932768
https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/7932769
https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/7932770

They all failed after a retry also

**Comment by pepoviola** [comment]
closing here, follow-up in #8944
